Setup.lhs

```diff
@@ -72,33 +53,10 @@ generateBuildModule verbosity pkg lbi = do
         , ""
         , "compiler :: String"
         , "compiler = " ++ (show $ showCompilerId $ compiler lbi)
-        , ""
-        , "depsFlag :: String"
-#if MIN_VERSION_Cabal(1,24,0)
-        , "depsFlag = " ++ show "-package-id="
-#else
-        , "depsFlag = " ++ show "-package="
-#endif
```

Comment:
Why don't we need to be picky about choosing `-package=` vs. `=package-id` anymore? (We seem to be unconditionally using  `=package-id` in `doctests.hsc`.)